tests/2017-10-01/Microsoft.Storage.tests.json

```diff
@@ -0,0 +1,105 @@
+{
+    "tests": [
```

Comment:
>     [](start = 0, length = 4)

please format the test file with consistent indents.

Reply:
Sure, just a minute.